test/unit-tests/org/orgTest.js

```diff
@@ -1,15 +1,21 @@
 const server = require('../../../test-utils/index')
 const chai = require('chai')
 const expect = chai.expect
+const assert = chai.assert
 chai.use(require('chai-http'))
 const mongoose = require('mongoose')
 const headers = require('./mockObjects.org').headers
 const existentOrg = require('./mockObjects.org').existentOrg
 const nonExistentOrg = require('./mockObjects.org').nonExistentOrg
+const orgWithZeroIdQuota = require('./mockObjects.org').orgWithZeroIdQuota
+const orgWithNegativeIdQuota = require('./mockObjects.org').orgWithNegativeIdQuota
+const secretariatHeader = require('../cve-id/mockObjects.cve-id').secretariatHeader
```

Comment:
Remove if not being used

Reply:
Removing `secretariatHeader`, the other 2 on line 10-11 are using used.